app/controllers/admin/calendar_events_controller.rb

```diff
@@ -8,6 +8,16 @@ def index
       project = Project.find(params[:project_id])
       authorize project, :show?
       project_filter = {id: project.id}
+    elsif params[:person_id]
+      person = Person.find(params[:person_id])
+      authorize person
+      authorize Project
+
+      project_ids = []
+      person.agent_projects.each do |project|
+        project_ids.push(project.id)
+      end
+      project_filter = {id: project_ids}
```

Comment:
Here is a hint for how to simplify the above and incorporate policy_scope as we discussed:

```
project_filter = {id: policy_scope(person.agent_projects).pluck(:id)}
```

Please please please let me know if how this works this isn't clear. You may need to look up `pluck`. Please understand why using `pluck` is better than using `.map(&:id)` in this case. Feel free to use some mentoring time here.